homeassistant/components/notify/clicksend.py

```diff
@@ -0,0 +1,71 @@
+"""
+Clicksend platform for notify component.
+For more details about this platform, please refer to the documentation at
+https://clicksend.com/help
```

Comment:
The link will be https://home-assistant.io/components/notify.clicksend/ for the platform documentation.

If the configuration is not passing the validation then a log entry is created which is composed on-the-fly and takes the platform name into account.